README.md

```diff
@@ -85,6 +85,40 @@ In addition, you should ensure your application's session cookie domain configur
 
     'domain' => '.domain.com',
 
+### Broadcasting auth
+
+In order for your app to occupy a private channel, please follow the instructions to [Configure Broadcasting in Laravel](https://laravel.com/docs/6.x/broadcasting#configuration).
+Once complete, please verify that the Broadcast route is protected only by the `web` middleware. You must not specify the `auth:airlock` middleware here.
```

Comment:
I'm not sure why this would be required? Can you explain more what problems just using auth:airlock gave you?